code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1125,7 +1125,7 @@
 	build_type = MECHFAB
```

Comment:
```suggestion
	build_type = PROTOLATHE | MECHFAB
```
Ok but why not just make it available to both?

Reply:
It would then become the only borg module available through the protolathe.

Janitor already has to visit robotics in order to get the VTEC for the janicart anyway (using buffer without it is pain). Still has reason to visit RnD for the other janitorial gear they can provide.

Reply:
I mean it'd keep the behaviour as it used to be, and this is also a piece of janitor equipment which is mostly available in RnD, and they'd still have the VTEC as a reason to go to robotics

Reply:
I could go either way, really. Keeping existing behavior is nice, but it does make a bit more sense to have it be mechfab only. Up to CRUNCH, IMO.

Reply:
IMHO it's not a very drastic change for it to only be available from the mechfab (for above reasons). It'd feel very weird to have a singular borg upgrade in the protolathe.

Either way, Jani is going to be visiting both of the science desks.